tests/test_serializers.py

```diff
@@ -108,22 +123,44 @@ def test_property_serialized_correctly(self):
         self.assertEqual(self.serializer.data, expected_result)
 
 
-class DeserializeSingleProperty(TestCase):
-    @patch('rest_framework.relations.RelatedField.get_queryset')
-    @patch.object(PreferenceForWrite, 'objects')
-    @patch.object(PreferenceForWrite, 'save')
-    def setUp(self, save_mock, manager_mock, get_queryset_mock):
-        # This emulates property in the database, which needs to be updated
-        # as a result of given test.
-        self.existing_prop_mock = Mock(name='name', value='value')
+class MockDBMixin(object):
```

Comment:
D209 Multi-line docstring closing quotes should be on a separate line
D210 No whitespaces allowed surrounding docstring text